ister.py

```diff
@@ -380,6 +380,10 @@ def copy_os(args, template, target_dir):
     swupd_command += " --contenturl={0}".format(args.contenturl)
     swupd_command += " --versionurl={0}".format(args.versionurl)
     swupd_command += " --format={0}".format(args.format)
+
+    if args.state is not None:
```

Comment:
So looking at this a little closer I should have reviewed the patch I sent you better. I think instead of None, args.state should default to '/var/lib/swupd'. The reason for this is to fit in with the rest of the options passed for contenturl, versionurl and format better.